### ISSUE TITLE
Inference netg_4000 fine tuning, results are black images

### ISSUE THREAD

**Issue by mindFlayer123** [xinntao/Real-ESRGAN#574]
Hi, I finetuned the model using `[finetune_realesrgan_x4plus_pairdata.yml]` for about 5000 iterations and some other settings like:
```num_worker_per_gpu: 2
  batch_size_per_gpu: 1
```
setting learning rate from 1e-4 to 1e-3 and changing this:
```
 scheduler:
    milestones: [4000]

  total_iter: 4000
```
Rest as it is.
Total epochs count was 3 only, as I was using colab I done these settings for faster training. But, after obtaining the new weights when I tested with inputs, the results I got were complete black images. What could be the possible issue ? Training eta : 22 mins.
@xinntao

**Comment by yulongblue** [comment]
I am also experiencing the same problem. Have you solved it?
